common/test/acceptance/pages/studio/settings_advanced.py

```diff
@@ -227,5 +227,6 @@ def expected_settings_names(self):
             'instructor_info',
             'create_zendesk_tickets',
             'ccx_connector',
-            'enable_ccx'
+            'enable_ccx',
+            'enable_subsection_grades_saved',
```

Comment:
Trailing commas are good for multi-line collections and function calls that take a variable number or order of arguments.  Clean diffs are nice, but not that important.  I like them more because they prevent frustration when you add arguments or reorder them, and forget to fix the commas.